lib/bolt/plugin.rb

```diff
@@ -151,7 +151,8 @@ def initialize(config, pal, analytics = Bolt::Analytics::NoopClient.new, load_pl
         msg = "Configuration for the PuppetDB plugin must be in the 'puppetdb' config section, not 'plugins'"
         raise Bolt::Error.new(msg, 'bolt/plugin-error')
       end
-      @unresolved_plugin_configs['puppetdb'] = config.puppetdb if config.puppetdb
+      @unresolved_plugin_configs['puppetdb'] = config.puppetdb.merge('default'   => config.default_puppetdb,
```

Comment:
What was that `if config.puppetdb` guarding against? Is there a case that could be `nil`?